app/controllers/courses_controller.rb

```diff
@@ -0,0 +1,71 @@
+class CoursesController < ApplicationController
+  layout 'dashboard'
+  before_action :authenticate_user!
+
+  def new_course
+    @page_title = 'Dashboard -> Add a course'
```

Comment:
use constant

Reply:
I removed all of the hard-coded string in [Implement-Schedule-Mail-Send branch](https://github.com/sks-wtag/LMS/pull/9/files#diff-6587ddd8f759a5e02861d5421da0cf20bdfd264f22b6b3ab83e1caff3b154973).